lml/src/main/java/com/github/czyzby/lml/parser/impl/attribute/group/DebugRecursivelyLmlAttribute.java

```diff
@@ -2,21 +2,37 @@
 
 import com.badlogic.gdx.scenes.scene2d.Group;
 import com.github.czyzby.lml.parser.LmlParser;
+import com.github.czyzby.lml.parser.action.ActorConsumer;
 import com.github.czyzby.lml.parser.tag.LmlAttribute;
 import com.github.czyzby.lml.parser.tag.LmlTag;
+import com.github.czyzby.lml.util.LmlUtilities;
 
 /** See {@link Group#setDebug(boolean, boolean)}. Second method argument is always true, so this method sets debug for
  * all children; for non-recursive debug, use default debug attribute. Mapped to "debugRecursively".
  *
  * @author MJ */
 public class DebugRecursivelyLmlAttribute implements LmlAttribute<Group> {
+    private static class DebugAction implements ActorConsumer<Object, Object> {
```

Comment:
Second generic parameter is used as an argument in `consume`. So I could use `<Void, Group>`, but `LmlUserObject#addOnCloseAction` is requiring `ActorConsumer<?, Object>` so I have to use `Void, Object` but I think `<Object, Object>` looks more clear.